opensquirrel/passes/router/shortest_path_router.py

```diff
@@ -31,9 +43,9 @@ def route(self, ir: IR) -> IR:
                 if not graph.has_edge(q0.index, q1.index):
                     try:
                         shortest_path = nx.shortest_path(graph, source=q0.index, target=q1.index)
-                        for start_qubit_index, end_qubit_index in zip(shortest_path[:-1], shortest_path[1:]):
-                            ir.statements.insert(statement_index, SWAP(start_qubit_index, end_qubit_index))
-                            statement_index += 1
+                        num_swaps_inserted = len(shortest_path) - 2
```

Comment:
Perhaps add a comment, why minus 2, _e.g._:
```suggestion
                        num_swaps_inserted = len(shortest_path) - 2  # the factor 2 is there take care of the start and end nodes.
```
Also, apply this to the A* router.